pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -593,24 +594,25 @@ class Build {
         }
 
         def appOptions="customJtx=${excludeRoot}/jenkins/jck_run/jdk${jdkVersion}/${excludePlat}/temurin.jtx"
-
+        def targets
         if (configureArguments.contains('--enable-headless-only=yes')) {
             // Headless platforms have no auto-manuals, so do not exclude any tests
             appOptions=""
+            targets = ['serial': 'sanity.jck,extended.jck,special.jck']
+        } else {
+            targets = ['serial': 'sanity.jck,extended.jck,special.jck,dev.jck']
         }
```

Comment:
For all these change dev.jck to be in its own target, as we can't run other targets on the especial Windows node
```
targets['serial_dev'] = 'dev.jck'
```